eidas-saml-parser/src/test/java/uk/gov/ida/notification/eidassaml/IstioHeaderTest.java

```diff
@@ -14,18 +14,16 @@
 import javax.ws.rs.core.Response;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.ida.notification.shared.IstioHeaders.X_B3_FLAGS;
+import static uk.gov.ida.notification.shared.IstioHeaders.X_B3_PARENTSPANID;
+import static uk.gov.ida.notification.shared.IstioHeaders.X_B3_SAMPLED;
+import static uk.gov.ida.notification.shared.IstioHeaders.X_B3_SPANID;
+import static uk.gov.ida.notification.shared.IstioHeaders.X_B3_TRACEID;
+import static uk.gov.ida.notification.shared.IstioHeaders.X_OT_SPAN_CONTEXT;
+import static uk.gov.ida.notification.shared.IstioHeaders.X_REQUEST_ID;
 
 public class IstioHeaderTest extends EidasSamlParserAppRuleTestBase {
```

Comment:
As these are apprule tests, is it worth moving them to the apprule package?

Reply:
Or even adding the test to the `EspAuthnRequstAppRuleTest` class?

Reply:
Gonna split this out into a new ticket.